license/open_data_commons_by_1.yaml

```
@@ -1,3 +1,4 @@
 title: 'Open Data Commons Attribution License (ODC-By) v1.0'
+title_short: 'ODC-By v.10'
```

Correcting a typo carried through from #5 

```suggestion
title_short: 'ODC-By v1.0'
```